### PR TITLE
Various of additions and changes

### LINKED ISSUE #18: Add converse.js
Add converse.js as webclient recommendation

### PR DESCRIPTION
**Pull request by sjehuda**
Fixes #18 #27 #30 #31

I need help with Javascript. See below: https://github.com/ge0rg/easy-xmpp-invitation/pull/32#issuecomment-1130000347

### CODE REVIEW

File: lang/en.json [hunk 1]
```diff
@@ -11,9 +11,9 @@
     "button": "Join the chat room {{name}}",
     "":""
   },
-  "clients": "If this link does not work, you need to install and configure an XMPP client, and visit this page again afterwards.",
-  "recommend": "We recommend one of:",
-  "checkfulllist": "Check the <a href='https://xmpp.org/software/clients.html'>full list of XMPP clients</a> for other platforms.",
-  "xmppis": "XMPP is a provider independent form of instant messaging.  You can pick from many different clients and have a free choice of server operators to participate.",
+  "clients": "You need install an XMPP client, to access the XMPP network.",
```
Comment (licaon-kter) [inline lang/en.json:14]: "need TO install"

Comment (sjehuda) [reply lang/en.json:14]: Thank you! It will be taken care of

**Comment by sjehuda** [comment]
OSX Consideration: Replace Psi by Beagle IM.

Link
https://beagle.im/

Logo
https://raw.githubusercontent.com/tigase/beagle-im/master/BeagleIM/Assets.xcassets/AppIcon.appiconset/IMG_0720_1024.png

**Comment by sjehuda** [comment]
> Fixes #18 #27 #30 #31
> 
> I need help with `i.css`
> 
> ```
> .img-center {
> 	display: flex;
> 	/* FIXME Why margin: auto; is not equal on all sides?
> 	margin-right: 8%;
> 	margin-left: 8%; */
> 	width: 3%;
> }
> ```

Using `border: solid green` shows that it's not a problems with CSS.
The problem is in Javascript, it appears.

I don't see what's wrog but my guess is that it has to do with the function `function load_clients(url)` which uses `XMLHttpRequest` and maybe too fast for the elements to properly place themselves, or I'm not doing something correctly.

```javascript
	function show_clients(client_array) {
		var list = document.getElementById('client_list');
		for (var id = 0; id < client_array.length; id++) {
			var item = document.createElement('span');
			item.innerHTML = client_array[id];
			list.appendChild(item);
		}
	}
```

```html
<p class="lead img-center text-center" id="client_list">
 <span>
  <a href="https://play.google.com/store/apps/details?id=org.yaxim.androidclient">
  <img src="assets/yaxim.svg" height="60">
  yaxim
  </a>
 </span>
 <span>
  <a href="https://play.google.com/store/apps/details?id=eu.siacs.conversations">
  <img src="assets/conversations.svg" height="60">
  Conversations
  </a>
 </span>
 <span>
  <a href="https://play.google.com/store/apps/details?id=com.xabber.android.beta">
  <img src="assets/xabber.svg" height="60">
  Xabber
  </a>
 </span>
 <span>
  <a href="https://play.google.com/store/apps/details?id=org.yaxim.bruno">
  <img src="assets/bruno.png" height="60">
  Bruno
  </a>
 </span>
 <span>
  <a href="https://play.google.com/store/apps/details?id=org.snikket.android">
  <img src="assets/snikket.svg" height="60">
  Snikket
  </a>
 </span>
</p>
```
I've changed `list.appendChild(item);` by `list.append(item)` and `list.prepend(item)` and it didn't change anything.

simplescreenrecorder-2022-05-18_15 58 51 mp4

Video: https://user-images.githubusercontent.com/19249089/169046658-78d97e41-52f4-4015-93ba-d80f24c1f54f.mp4

**Comment by sjehuda** [comment]
> We have started using DOAP files

XML! Just as I like it!
You've a business with an XML fan, who has very recently ported New Feed Preview feature into a [Userscript](https://openuserjs.org/scripts/sjehuda/Newspaper) ;)
Can you refer me to example code?

> Maybe this helps your approach. Perhaps, this also triggers your interest to support at other corners of the network! It would be highly apprecitated! Let me know.

I'm all for it! XMPP has served me for over a decade.
I'd be more than glad to give something (or things) back!

My email is sjehuda@yandex.com

Btw, @Echolon can you refer a JS savvy to this ticket.  I realy want to remove the Draft tag from this PR and make it ready.  I'll take care of DOAP just after we done with this PR.

**Comment by Echolon** [comment]
@sjehuda Alright, feel free to join commteam@muc.xmpp.org to have a chat.

Find the website shortcode here:
https://github.com/xsf/xmpp.org/blob/master/themes/xmpp.org/layouts/shortcodes/software-list-doap.html

**Comment by sjehuda** [comment]
Marked pull request as ready for review.
Special thank you to @asdofindia who has provided this solution within an hour after my public call for help.
Also, thank you to @horazont of search.jabber.network who has referred me to this project upon my request.

**Comment by sjehuda** [comment]
This PR is ready.
Next step. convert JSON files into DOAP (XML) files.  Each client has its own file DOAP file.

**Comment by ge0rg** [comment]
Thank you very much, I've slightly-squashed, reordered and merged this PR in c035b7d1ea3c7091a7bf4ed20292e0c70508e6bb.

**Comment by sjehuda** [comment]
Thank you, George!  You're awesome, bro!

**Pull request closed without merging**